### PR TITLE
Update SIG Instrumentation leads

### PR DESCRIPTION
**Pull request by ehashman**
Removing @piosz, and adding myself and @logicalhan.

/cc @brancz 

See
- https://groups.google.com/d/msg/kubernetes-sig-instrumentation/IZWt4loyx7M/svIYpz0kCwAJ
- https://docs.google.com/document/d/1FE4AQ8B49fYbKhfg4Tx0cui1V0eI4o3PxoqQPUwNEiU/edit

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1330,12 +1330,16 @@ sigs:
...
+    - github: ehashman
+      name: Elana Hashman
+      company: Red Hat
+    - github: logicalhan
+      name: Han Kang
+      company: Google
+    tech_leads:
     - github: brancz
       name: Frederic Branczyk
       company: Red Hat
```
Comment (logicalhan) [inline sigs.yaml]: Let's add @piosz as emeritus_lead.

```
emeritus_leads:
    - github: piosz
      name: Piotr Szczesniak
      company: Google
```

Comment (ehashman) [reply sigs.yaml]: Did not include the company because other emeritus_leads in here didn't include that.

**Comment by cblecker** [comment]
/approve
LGTM to @brancz

**Comment by brancz** [comment]
/lgtm

Awesome to have you!!! 🎉🎊

**Pull request merged**